src/components/ecl-button/ecl-button.stories.ts

```diff
@@ -35,6 +38,8 @@ const getArgTypes = () => {
       name: 'Icon only',
       type: { name: 'boolean'},
       description: 'Hides the label (not for screen-readers)',
+      defaultValue: false,
```

Comment:
i didn't notice this, this is replaced by the args, the ones we return from getArgs()

Reply:
i fixed the issues, added the new variants and controls. also disabled.